text/0000-dual-light-shadow.md

```diff
@@ -129,6 +130,54 @@ If `"dual"` is used in this context, a compile-time error is thrown:
 </template>
 ```
 
+Only three values are accepted in this context: `"light"`, `"shadow"`, or `"inherit"`. Any other value will throw a compile-time error.
+
+### Inheritance
+
+`"inherit"` is a special value of `lwc:render-mode` that can only be used in the context of a child LWC component:
+
+```html
+<template lwc:render-mode="dual">
+    <x-component lwc:render-mode="inherit"></x-component>
+</template>
+```
+
+This instructs the `<x-component>` to render using whatever mode the parent component is using – either shadow or light. If `<x-component>` is not a dual-mode component, then a runtime error will be thrown.
+
+For the parent component, however, `"inherit"` can be used in non-dual-mode components as well as dual-mode components:
+
+```html
+<template lwc:render-mode="light">
+    <!-- x-component will render as light -->
+    <x-component lwc:render-mode="inherit"></x-component>
+</template>
+```
+
+In this case, the same rules apply – the child component uses the mode inherited from its parent.
+
+The above example is equivalent to:
+
+```html
+<template lwc:render-mode="light">
+    <!-- x-component will render as light -->
+    <x-component lwc:render-mode="light"></x-component>
+</template>
+```
+
+For the purposes of inheritance, `<slot>` contents are considered to be children of the slotting component, not the
+slottable component.
+
+```html
+<template>
+    <x-slottable>
+        <!-- Mode is inherited from this template, not from x-slottable's template -->
+        <x-component lwc:render-mode="inherit"></x-component>

```

Comment:
in this example, who dictates the mode? how can an author look at this code an know? what is the process of knowing what mode this is going to run on?

Reply:
> in this example, who dictates the mode?

The top-level `<template>` on Line 171 dictates the mode. In this case, it will be shadow mode, because there is no `render-mode` on line 171.

> how can an author look at this code an know?

They would need to glance up to the top `<template>` in the template they are currently authoring. 🙂 

> what is the process of knowing what mode this is going to run on?

Again, glance up top. 🙂 And if `<x-component>` does not support dual mode, you will get an error.